vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -77,7 +79,23 @@ func NewVolumeDriver(port int, useMockEsx bool, mountDir string, driverName stri
 		}
 	}
 
-	d.refCounts.Init(d, mountDir, driverName)
+	err := d.refCounts.Init(d, mountDir, driverName)
+	// If refcounting wasn't successfull, schedule one again
+	if err != nil {
```

Comment:
The rescheduling of the refcount scan seems like a functionality thats owned by the refcounting module. The VMDK driver shouldn't have to care what the refcount module does to handle refcount init failures. It just calls refcnt.Init() and that handles any retry mechanism. In fact that way refcount module has more flexibility on handlng scan retries than if Init() is re-done from here.